maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FdcItemsEntity.java

```diff
@@ -0,0 +1,83 @@
+package gov.uk.courtdata.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import jakarta.xml.bind.annotation.XmlType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "FDC_ITEMS", schema = "TOGDATA")
+@XmlType
+public class FdcItemsEntity {
+
+    @Id
+    @SequenceGenerator(name = "fdc_contributions_gen_seq", sequenceName = "S_GENERAL_SEQUENCE", allocationSize = 1, schema = "TOGDATA")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "fdc_contributions_gen_seq")
+    @Column(name = "ID", nullable = false)
+    private Integer id;
```

Comment:
Please double check these fields against the DDL from the DB, i.e. the max length/size matches that in the prod DB